duckduckgo/BrowserServicesKit
# Improve file creation

Task/Issue URL: https://app.asana.com/0/1200194497630846/1203942293612017/f
iOS PR:  n/a
macOS PR: n/a
What kind of version bump will this require?: n/a

**Description**:
Fixes a tedious process of creating new files.

1. Introduce a new file creation template called "Swift File For Package" and an instruction how to symlink it to your Xcode's Template directory.
2. Introduce a copyright template placed inside .swiftpm in a xcshareddata subdir.

**Steps to test this PR**:
1. Follow the updated instructions from README.md.
2. Press Command+N and select Swift File For Package
3. Select a name and confirm the file has been named correctly as well as the copyright header is present.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
